api/v1alpha1/managedcertificate_types.go

```diff
@@ -0,0 +1,108 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Important: Run "make crd" to regenerate code after modifying this file
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ManagedCertificateSpec defines the desired state of ManagedCertificate.
+type ManagedCertificateSpec struct {
+	// Target defines the targets that the Certificate will be bound to.
```

Comment:
nit: target here, singular?

Reply:
Plural on purpose. There will be other targets in the future (mounting directly to Istio Ingress)